lib/java/com/google/android/material/navigationrail/NavigationRailMenuView.java

```diff
@@ -63,9 +63,8 @@ protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
     }
 
     // Set view to use parent width, but wrap all item heights
-    int parentWidth = MeasureSpec.getSize(widthMeasureSpec);
     setMeasuredDimension(
-        View.resolveSizeAndState(parentWidth, widthMeasureSpec, /* childMeasuredState= */ 0),
+        MeasureSpec.getSize(widthMeasureSpec),
         View.resolveSizeAndState(measuredHeight, heightMeasureSpec, /* childMeasuredState= */ 0));
```

Comment:
Can just be `View.resolveSize(measuredHeight, heightMeasureSpec)`?

Reply:
No, because the `MEASURED_STATE_TOO_SMALL` flag will be lost.

Reply:
Ah you are right. : )